doc/parameters/param_list.md

```diff
@@ -102,328 +101,332 @@ Namespace: /parent_ns/local_ns
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.sum_scores | false | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.sum_scores | false | Whether to allow for scores to be sumed up |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### prefer_forward TrajectoryCritic
+## prefer_forward TrajectoryCritic
 
 * `<name>`: prefer_forward critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.penalty | 1.0 | |
-| `<dwb plugin>`.`<name>`.strafe_x | 0.1 | |
-| `<dwb plugin>`.`<name>`.strafe_theta | 0.2 | |
-| `<dwb plugin>`.`<name>`.theta_scale | 10.0 | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.penalty | 1.0 | Penalty to apply to backward motion |
+| `<dwb plugin>`.`<name>`.strafe_x | 0.1 | Minimum X velocity before penalty |
+| `<dwb plugin>`.`<name>`.strafe_theta | 0.2 | Minimum angular velocity before penalty |
+| `<dwb plugin>`.`<name>`.theta_scale | 10.0 | Weight for angular velocity component |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### twirling TrajectoryCritic
+## twirling TrajectoryCritic
 
 * `<name>`: twirling critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.scale | 0.0 | |
+| `<dwb plugin>`.`<name>`.scale | 0.0 | Weight scale |
 
-### goal_align TrajectoryCritic
+## goal_align TrajectoryCritic
 
 * `<name>`: goal_align critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.forward_point_distance | 0.325 | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.forward_point_distance | 0.325 | Point in front of robot to look ahead to compute angular change from |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### map_grid TrajectoryCritic
+## map_grid TrajectoryCritic
 
 * `<name>`: map_grid critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.aggregation_type | "last" | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.aggregation_type | "last" | last, sum, or product combination methods |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### path_dist TrajectoryCritic
+## path_dist TrajectoryCritic
 
 * `<name>`: path_dist critic name defined in `<dwb plugin>.critics`
 
 | Parameter | Default | Description |
 | ----------| --------| ------------|
-| `<dwb plugin>`.`<name>`.aggregation_type | "last" | |
-| `<dwb plugin>`.`<name>`.scale | 1.0 | |
+| `<dwb plugin>`.`<name>`.aggregation_type | "last" | last, sum, or product combination methods |
+| `<dwb plugin>`.`<name>`.scale | 1.0 | Weight scale |
 
-### path_align TrajectoryCritic
+## path_align TrajectoryCritic
```

Comment:
PathAlignCritic inherits from PathDistCritic which inherits from MapGridCritic. 

If we are including parameters from parent classes, then PathAlignCritic should also include the aggregation_type parameter (from MapGridCritic) 

| `<dwb plugin>`.`<name>`.aggregation_type | "last" | last, sum, or product combination methods |